docs/src/main/asciidoc/amqp.adoc

```diff
@@ -295,15 +271,50 @@ Then, you just need to run the application using:
 
 Open `http://localhost:8080/prices.html` in your browser.
 
-NOTE: If you started the AMQP broker with docker compose, stop it using `CTRL+C` followed by `docker-compose down`.
+== Running in JVM or Native mode
+
+When not running in dev or test mode, you will need to start your AMQP broker.
+You can follow the instructions from the https://activemq.apache.org/components/artemis/documentation/[Apache ActiveMQ Artemis] or create a `docker-compose.yaml` file with the following content:
+
+[source, yaml]
+----
+# A docker compose file to start an Artemis AMQP broker
+version: '2'
+
+services:
+
+  artemis:
+    image: quay.io/artemiscloud/activemq-artemis-broker:0.1.2
+    ports:
+      - "8161:8161"
+      - "61616:61616"
+      - "5672:5672"
+    environment:
+      AMQ_USER: quarkus
+      AMQ_PASSWORD: quarkus
+----
+
+Once created, run `docker-compose up`.
+
+NOTE: This is a development broker, do not use in production.
+
+You can build and run the application in JVM mode with:
+
+[source, bash]
+----
+./mvnw package
+java -jar target/quarkus-app/quarkus-run.jar -Damqp-user=quarkus -Damqp-password=quarkus
```

Comment:
Just realized -- in this case, `-Damqp-user=quarkus -Damqp-password=quarkus` needs to go _before_ `-jar`, otherwise they are ignored.

Reply:
Incidentally, this is done correctly in the `NOTE` just a few lines below :-)

Reply:
Great catch, I fixed it.